src/chains/ethereum/options/src/chain-options.ts

```diff
@@ -196,7 +198,12 @@ export const ChainOptions: Definitions<ChainConfig> = {
   hardfork: {
     normalize,
     cliDescription: "Set the hardfork rules for the EVM.",
-    default: () => "london",
+    default: () => {
+      const forkArg = yargs.parse(process.argv)['fork']
```

Comment:
As a general comment: There's a couple of code lines missing trailing ; - if you install `Prettier - Code formatter` in VS Code, and enable "formatOnSave" (https://www.robinwieruch.de/how-to-use-prettier-vscode/), it'll standardise styling for you every time you save.